subjects/devops/cloud-kube/README.md

```diff
@@ -0,0 +1,211 @@
+
+### Hints
+
+Before starting this project, you should know the following:
+
+- Basic DevOps concepts and practices.
+- Familiarity with containerization and orchestration tools, such as Docker and
+  Kubernetes.
+- Understanding of your chosen Cloud provider platform.
+- Familiarity with Terraform as a Infrastructure as Code (IaC) tools.
+- Knowledge of monitoring and logging tools, such as Prometheus, Grafana, and
+  ELK stack.
+
+> Any lack of understanding of the concepts of this project may affect the
+> difficulty of future projects, take your time to understand all concepts.
+
+> Be curious and never stop searching!
+
+### Role play
+
+To enhance the learning experience and assess your knowledge, a role play
+question session will be included as part of the `Cloud-Kube` Project.
+This section will involve answering a series of questions in a simulated
+real-world scenario where you assume the role of a Cloud engineer explaining
+your solution to a team or stakeholder.
+
+The goal of the role play question session is to:
+
+- Assess your understanding of the concepts and technologies used in the
+  project.
+- Test your ability to communicate effectively and explain your decisions.
+- Challenge you to think critically about your solution and consider
+  alternative approaches.
+
+Prepare for a role play question session where you will assume the role of a
+Cloud engineer presenting your solution to your team or a stakeholder. You
+should be ready to answer questions and provide explanations about your
+decisions, architecture, and implementation.
+
+### Architecture
+
+By using the [provided container
+images](https://github.com/01-edu/play-with-containers) you have to define your
+K8s manifests and deploy your infrastructure on your chosen cloud
+provider respecting the project requirements, consisting of the following
+components:
+
+- `inventory-database container` is a PostgreSQL database server that contains
+  your inventory database, it must be accessible via port `5432`.
+- `billing-database container` is a PostgreSQL database server that contains
+  your billing database, it must be accessible via port `5432`.
+- `inventory-app container` is a server that contains your
+  inventory-app code running and connected to the inventory database and
+  accessible via port `8080`.
+- `billing-app container` is a server that contains your billing-app
+  code running and connected to the billing database and consuming the messages
+  from the RabbitMQ queue, and it can be accessed via port `8080`.
+- `RabbitMQ container` is a RabbitMQ server that contains the queue.
+- `api-gateway-app container` is a server that contains your
+  API Gateway code running and forwarding the requests to the other
+  services, and it's accessible via port `3000`.
+
+Design the architecture for your cloud-based microservices' application. You
+are free to choose the services and architectural patterns that best suit your
```

Comment:
we should force the student to use a k8s cloud service

Reply:
I will add some more details to ask to use K8s in their microservices and to structure their project based on this choice.